### PR TITLE
Real srs

### PR DESCRIPTION
**Pull request by swasilyev**
zcash stuff hardcoded

### CODE REVIEW

File: bandersnatch_vrfs/src/ring.rs [hunk 1]
```diff
@@ -28,10 +29,17 @@ pub type RingVerifier = ring::ring_verifier::RingVerifier<Fq, RealKZG, SWConfig>
 pub type ProverKey = ring::ProverKey<Fq, RealKZG, SWAffine>;
 pub type VerifierKey = ring::VerifierKey<Fq, RealKZG>;
 
+// A point on Jubjub, not belonging to the prime order subgroup.
+// Used as the point to start summation from, as inf doesn't have an affine representation.
+const COMPLEMENT_POINT: crate::Jubjub = {
+    const X: Fq = Fq::ZERO;
+    const Y: Fq = MontFp!("11982629110561008531870698410380659621661946968466267969586599013782997959645");
+    crate::Jubjub::new_unchecked(X, Y)
+};
```
Comment (burdges) [inline bandersnatch_vrfs/src/ring.rs:42]: Yay

**Comment by burdges** [comment]
I looks good.  Added a commit which hopefully fixes no std.  We'll see if CI catches anything else.

**Comment by swasilyev** [comment]
Thanks

**Review by davxy — APPROVED** [review]
Great

**Comment by burdges** [comment]
We'd planned on merging this in January once your back @davxy  And sergey depended upon this branch in https://github.com/paritytech/verifiable/pull/2 so everything works for now.

We could merge this now though, if you doubt merging messes anything up for other substrate forks, but really no rush..

**Comment by burdges** [comment]
https://github.com/w3f/ring-vrf/pull/82 is merged

**Pull request merged**